docs/resources/azure_redis_caches.md

```diff
@@ -0,0 +1,118 @@
+---
+title: About the azure_redis_caches Resource
+platform: azure
+---
+
+# azure_redis_caches
+
+Use the `azure_redis_caches` InSpec audit resource to test properties related to Azure Redis Cache for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_redis_caches` resource block returns all Azure Redis Caches within a Resource Group.
+```ruby
+describe azure_redis_caches(resource_group: 'my-rg') do
+  #...
+end
+```
+or
+```ruby
+describe azure_redis_caches(resource_group: 'my-rg') do
+  #...
+end
+```
+## Parameters
+
+The parameter should be provided for valid query
+- `resource_group`
+
+| Name                            | Description                                                                      |
+|---------------------------------|----------------------------------------------------------------------------------|
+| resource_group                  | Azure resource group that the targeted resource resides in. `MyResourceGroup`    |
+
+## Properties
+
+|Property            | Description                                        | Filter Criteria<superscript>*</superscript> |
+|--------------------|----------------------------------------------------|-----------------|
+| ids                | A list of the unique resource ids.                 | `id`            |
+| names              | A list of name for all the Resource names.         | `name`          |
+| types              | A list of types for all the resources.             | `type`          |
+| locations          | A list of locations for all the resources.         | `location`      |
+| properties         | A list of Properties all the resources.            | `properties`    |
```

Comment:
Are the filter criteria correct? I think properties should be `property`, `tags` should be `tag`,  `sku_capacitie` should be `sku_capacity`, there's a bunch of others that are plural but should be singular.

Reply:
Yeah got it ..

Reply:
Yeah got it ..

Reply:
so whatever is in fields [here](https://github.com/inspec/inspec-azure/pull/394/files#diff-dea98b49464579e78814f2989f3ab91799d558609ced436bbe1c6117b4de7ecfR27-R52) will represent the filter criteria in docs and the columns as property in docs .. 

But I did notice there were some discrepancies in the docs for the filter criteria and fixed it .. thanks for pointing out ..